tools/ratbagctl.1

```diff
@@ -1,72 +1,202 @@
-.TH ratbagctl 1 "Apr 13, 2016" ratbagctl
+.TH RATBAGCTL "1" "@version@"
 .SH NAME
 ratbagctl \- inspect and modify configurable mice
 .SH SYNOPSIS
-.B ratbagctl
-.RB [ \-h ]
-.RB [ \-V ]
-.br
-.B ratbagctl list-devices
-.br
-.B ratbagctl show-device
-.RI < device >
-.br
-.B ratbagctl show-profile
-.RI < device "> <" profile >
-.br
-.B ratbagctl show-resolution
-.RI < device "> <" profile "> <" resolution >
-.br
-.B ratbagctl show-button
-.RI < device "> <" profile "> <" button >
+.B ratbagctl [OPTIONS] {COMMAND} ... device
 .SH DESCRIPTION
-.SS Common options
-.TP
-.BR \-h
-displays a short help message (it can be used with any form of the
-.B ratbagctl
-command).
-.TP
-.BR \-V
-displays the version of
+.PP
+The
+.I ratbagctl
+tool queries or changes a device's settings.
+.PP
+This tool usually needs ratbagd to be running.
+.SH OPTIONS
+.TP 8
+.B \-\-version, \-V
+Show program's version number and exit.
+.TP 8
+.B \-\-verbose, \-v
+Print debugging output.
+.TP 8
+.B \-vvv
+Print more debugging output (with protocol output).
```

Comment:
this should just say something like 'Multiple -v options increase the verbosity`, because -vv is valid too